lib/travis/build/appliances/update_apt_keys.rb

```diff
@@ -22,8 +22,27 @@ def apply
               bash &>/dev/null
             KEYUPDATE
             sh.cmd command, echo: false
+
+            sh.cmd rabbit_key_update, echo: false
           end
         end
+
+        def rabbit_key_update
+          <<~EORABBIT
+          if [[ ( \"$TRAVIS_DIST\" == trusty || \"$TRAVIS_DIST\" == precise) && -f /etc/init.d/rabbitmq-server ]]; then
```

Comment:
Just to confirm, `precise` and `trusty` both use the `rabbitmq-trusty` key URL?

Reply:
Confirm. The contents of the following files is the same:
https://github.com/travis-ci/apt-source-safelist/blob/master/keys/rabbitmq-trusty.asc
https://github.com/travis-ci/apt-source-safelist/blob/master/keys/rabbitmq-precise.asc